modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptNestjsClientCodegen.java

```diff
@@ -265,6 +269,34 @@ private boolean isLanguageGenericType(String type) {
         return false;
     }
 
+    @Override
+    public List<CodegenParameter> fromRequestBodyToFormParameters(RequestBody body, Set<String> imports) {
```

Comment:
please separate this change of parameters into a separate PR. also, is there an easier way than to `extends CodegenParameter`?

Reply:
I was inpired by the typescript-fetch generator. If you know a better way to do it I can make the changes

Reply:
I see uses of `param.vendorExtensions` in others generators:

https://github.com/OpenAPITools/openapi-generator/blob/ed211050f1c9d9c45456c006ba1ded30271d1d91/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpNextgenClientCodegen.java#L204

Do you think it would be a better approach as it doesn't require to extend `CodegenParameter` ?

Reply:
oh that sounds compelling, yes please try if that works!

Reply:
Done ! It works as well 😃

Reply:
thanks!
are you motivated to also do that for the typescript-fetch generator in a separate PR?

Reply:
You mean using `vendorExtensions` instead of extending `CodegenParameter` ? Or keeping original names in requestParameters interfaces ? Or both maybe ? 😃